simapp/simd/cmd/cmd_test.go

```diff
@@ -0,0 +1,23 @@
+package cmd_test
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/cosmos/cosmos-sdk/simapp/simd/cmd"
+	"github.com/cosmos/cosmos-sdk/x/genutil/client/cli"
+)
+
+func TestInitCmd(t *testing.T) {
+	rootCmd, _ := cmd.NewRootCmd()
+	rootCmd.SetArgs([]string{
```

Comment:
Shouldn't we be testing the init command directly?

Reply:
Well, we do test the init cmd separately in x/genutil, yet it didn't catch the bug in #7026.

Here, we also test the PersistentPreRunE part of the command.